.gitignore

```diff
@@ -21,3 +20,5 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+node_modules/
```

Comment:
- [ ] As mentioned in the first review, please remove all unneccessary files from your repo. You still have node modules on github. Please note that even if you've added the node modules to .gitignore, you still need to remove the commited node modules from gitub.